test/CMakeLists.txt

```diff
@@ -201,3 +204,9 @@ foreach (_file IN LISTS UNIT_TESTS)
     endif()
 
 endforeach()
+
+if (NOT _ONEDPL_PSTL_OFFLOAD STREQUAL off)
+    file(GLOB_RECURSE _test_file "pstl_offload/memory/usm_memory_replacement.cpp")
+    file(GLOB_RECURSE _test_file1 "pstl_offload/memory/usm_memory_replacement_native_alloc.cpp")
+    onedpl_add_test(${_test_file} true ${_test_file1} ${_test_file2})
```

Comment:
It seems like we don't have a make target named `usm_memory_replacement.pass`, now it is named `usm_memory_replacement`. I think that is why we see CI failures in PSTL offload mode. 
We need to either modify the CI configuration or replace the part with 2 TUs into the separate test, as I mentioned before. Lets discuss separately.

Reply:
It seems suffix pass is expected to be meaningful. At least is libc++ I see `*.verify.cpp` in addition to pass. So I may want to restore it.

Reply:
I think restoration of `.pass` suffix is ok, but in this case `usm_memory_replacement.pass` should be skipped in the main test processing loop in CMake to avoid issues with test target duplication.